students/Kristy_Martini/lesson05/mailroom.py

```diff
@@ -121,9 +119,9 @@ def prompt_user(newReport=None):
     choice = input("Please enter the number associated with your choice: ")
     
     try:
-        arg_dict[choice](newReport)
+        arg_dict[choice](new_report)
     except KeyError:
-        prompt_user(newReport)
+        prompt_user(new_report)
 
 if __name__ == "__main__":
     prompt_user()
```

Comment:
Getting new errors on this one - can't add donations or send letters.  Please take another look and let me know if you need assistance.